src/python/WMCore/REST/Main.py

```diff
@@ -449,34 +456,36 @@ def run(self):
         # to run the server proper.  The parent monitors the child, and if
         # it exits abnormally, restarts it, otherwise exits completely with
         # the child's exit code.
-        cherrypy.log("WATCHDOG: starting server daemon (pid %d)" % os.getpid())
-        while True:
-            serverpid = os.fork()
-            if not serverpid: break
-            signal.signal(signal.SIGINT, signal.SIG_IGN)
-            signal.signal(signal.SIGTERM, signal.SIG_IGN)
-            signal.signal(signal.SIGQUIT, signal.SIG_IGN)
-            (xpid, exitrc) = os.waitpid(serverpid, 0)
-            (exitcode, exitsigno, exitcore) = (exitrc >> 8, exitrc & 127, exitrc & 128)
-            retval = (exitsigno and ("signal %d" % exitsigno)) or str(exitcode)
-            retmsg = retval + ((exitcore and " (core dumped)") or "")
-            restart = (exitsigno > 0 and exitsigno not in (2, 3, 15))
-            cherrypy.log("WATCHDOG: server exited with exit code %s%s"
-                         % (retmsg, (restart and "... restarting") or ""))
-
-            if not restart:
-                sys.exit((exitsigno and 1) or exitcode)
-
-            for pidfile in glob("%s/*/*pid" % self.statedir):
-                if os.path.exists(pidfile):
-                    with open(pidfile) as fd:
-                        pid = int(fd.readline())
-                    os.remove(pidfile)
-                    cherrypy.log("WATCHDOG: killing slave server %d" % pid)
-                    try:
-                        os.kill(pid, 9)
-                    except:
-                        pass
+        daemonize = os.getenv('DONT_DAEMONIZE_REST','False') != 'True'
```

Comment:
same here, can you please add the missing white space before the `'False'`

Reply:
thanks for spotting those Alan. Should be OK now.